content/authorization/reference/userscenarios/_index.nb.md

```
@@ -346,15 +346,18 @@ Tjenesten **Rapportering aksjonærregister** har et oppsett som gir følgende ti
 
 Rett Revisisjon har kjøpt inn Maestro for å rapportere aksjonærregisteroppgaven
 
+### Forutsetninger
+- Rett revisjon har delt med Maestro hvilket kundeforhold de har for hver enkel kunde.
+
 ### Steg
 
 1. Maestro sender forespørsel om å opprette systembruker for klientforhold som krever *revisormedarbeider*
```

markdownlint-cli2 (0.17.2)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)